pyannote/audio/interactive/recipes/voice_activity_detection.py

```diff
@@ -0,0 +1,222 @@
+                        mode="constant",
+                        value=0,
+                    )
+                if pipeline is not None:
+                    st = focus.start - extend
+                    ed = focus.end + extend
+                    if focus.start == 0:
+                        st = 0
+                    elif focus.duration < chunk:
+                        ed = focus.end
+                    longFocus = Segment(st, ed)
+                    longWaveform, sr = raw_audio.crop(file, longFocus)
+                    speech: Annotation = pipeline(
+                        {"waveform": longWaveform, "sample_rate": sr}
+                    )
+                    task_audio_spans_before = to_audio_spans(speech)
+                    task_audio_spans = []
+                    for span in task_audio_spans_before:
+                        span["start"] -= extend
+                        span["end"] -= extend
+                        if span["start"] < 0:
+                            if span["end"] < 0:
+                                break
+                            else:
+                                span["start"] = 0
+                        if span["end"] > chunk:
+                            if span["start"] > chunk:
+                                break
+                            else:
+                                span["end"] = chunk
+                        task_audio_spans.append(span)
+                else:
+                    task_audio_spans = []
+                waveform = waveform.numpy().T
+                task_audio = to_base64(normalize(waveform), sample_rate=SAMPLE_RATE)
+
+                yield {
+                    "path": path,
+                    "text": task_text,
+                    "audio": task_audio,
+                    "audio_spans": task_audio_spans,
+                    "audio_spans_original": deepcopy(task_audio_spans),
+                    "chunk": {"start": focus.start, "end": focus.end},
+                    "meta": {
+                        "file": text,
+                        "start": f"{focus.start:.1f}",
+                        "end": f"{focus.end:.1f}",
+                    },
+                }
+
+
+@prodigy.recipe(
+    "audio.vad",
+    dataset=("Dataset to save annotations to", "positional", None, str),
+    source=(
+        "Data to annotate (file path or '-' to read from standard input)",
+        "positional",
+        None,
+        str,
+    ),
+    chunk=(
+        "split long audio files into shorter chunks of that many seconds each",
+        "option",
+        None,
+        float,
+    ),
+    pipeline=(
+        "Yaml configuration file from pretrained model",
+        "option",
+        None,
+        str,
+    ),
+    precision=("Cursor speed", "option", None, int),
+)
+def voice_activity_detection(
+    dataset: str,
+    source: Union[str, Iterable[dict]],
+    chunk: float = 10.0,
+    pipeline: Optional[str] = None,
+    precision: int = 40,
```

Comment:
100ms should be more than enough, by default.

```suggestion
    precision: int = 100,
```